docs/UrlRoutes.md

```diff
@@ -36,7 +36,6 @@ Sometimes a resource has a view with more details, so we can append `/details` t
 
 Basically the same URL as accessing a resource, but with an appended `/edit`
```

Comment:
I love the tab system, let's get rid of this whole section and just mention we use tabs for all kinds of `edit` for resources?